src/clj/matcher_combinators/midje.clj

```diff
@@ -86,3 +134,22 @@
          ::result/value  mismatch-val
          ::result/weight 1}))))
 
+(def match-equals
+  "match but using strict `equals` matching behavior for maps, even nested ones."
+  (match-with {clojure.lang.IPersistentMap matchers/equals}))
+
+(def match-roughly
```

Comment:
still need to implement this for `clojure.test`, but it is a little more tricky because it takes in another arg